lib/rake/circle.rake

```diff
@@ -82,8 +85,10 @@ namespace :circle do
       next
     end
 
-    if CircleUtils.tagged?(SKIP_UI_TESTS_TAG)
-      ChatClient.log "Commit message: '#{CircleUtils.circle_commit_message}' contains [#{SKIP_UI_TESTS_TAG}], skipping UI tests for this run."
+    # We used to default to running UI tests in every CircleCI build, but that contributes to high usage of
+    # donated SauceLabs resources.  We now default to NOT run UI Tests unless explicitly requested in the commit message.
+    unless CircleUtils.tagged?(RUN_UI_TESTS_TAG)
+      ChatClient.log "Commit message: '#{CircleUtils.circle_commit_message}' does not contain [#{RUN_UI_TESTS_TAG}], skipping UI tests for this run."
       next
```

Comment:
Is this next command still correct? Will this skip over the rest of the logic after this?